Lib/sysconfig/__init__.py

```diff
@@ -414,6 +414,13 @@ def _init_non_posix(vars):
     vars['BINDIR'] = os.path.dirname(_safe_realpath(sys.executable))
     vars['TZPATH'] = ''
 
+    abiflags = vars['abi_thread']
+    if vars['EXT_SUFFIX'].endswith('_d.'):
+        abiflags += 'd'
+    # Do not touch lower-cased `abiflags` here.
+    # sys.abiflags is absent on Windows. So vars['abiflags'] should be empty.
```

Comment:
```suggestion
    # It is used elsewhere to calculate paths that, on Windows, do not include any flags
```

This is the real reason, which is unrelated to `sys.abiflags`.